tests/test_read_ic.py

```diff
@@ -0,0 +1,48 @@
+from swiftsimio import load
+from swiftsimio import Writer
+from swiftsimio.units import cosmo_units
+
+import unyt
+import numpy as np
+
+
+def test_read():
```

Comment:
A better name than `test_read` would be nice.

Reply:
I hate naming things, hopefully test_reading_ic_units is better.